crates/uv-dispatch/src/lib.rs

```diff
@@ -397,6 +397,67 @@ impl<'a> BuildContext for BuildDispatch<'a> {
         .await?;
         Ok(builder)
     }
+
+    async fn direct_build<'data>(
+        &'data self,
+        source: &'data Path,
+        subdirectory: Option<&'data Path>,
+        output_dir: &'data Path,
+        build_kind: BuildKind,
+        version_id: Option<String>,
+    ) -> Result<Option<String>> {
```

Comment:
What is being returned here? And should it be a `PathBuf`?

Reply:
Oh okay, it's documented below on the trait.

Reply:
This initially happened because PEP 517 says that the build backend returns the filename as last line, and we passed that around as a string.

Reply:
More specifically, the location on disk may not be normalized the same as `WheelFilename` is, so for non-uv build backends we have to pass out a string and only parse later, i added comments.